app/routes/docs.client.samples.md

```diff
@@ -164,3 +167,192 @@ for message in consumer.consume(end[0].offset - start[0].offset, timeout=1):
             continue
     print(message.value())
 ```
+
+## HEALPix Sky Maps
+
+HEALPix stands for <b>H</b>ierarchical, <b>E</b>qual <b>A</b>rea, and iso-<b>L</b>atitude <b>Pix</b>elisation is a scheme for indexing positions on the unit sphere.
+For localization of events, the multi-messenger community uses the standard [HEALPix](https://healpix.sourceforge.io) format with the file extension `.fits.gz`, as well as multi-resolution HEALPix format with the file extension `.multiorder.fits`. The preferred format is the multi-resolution HEALPix format, that is only format explicitly listed in the GCN.
```

Comment:
'.gz' isn't specific to a healpix file, it's just showing that it's zipped.
Also, what is meant by "listed in the GCN"?

Reply:
No, it's not specific to HEALPix, but the convention for localization FITS files is that it's either a flat-resolution file and gzip-compressed, or a multi-order file and _not_ compressed.